src/main/resources/assets/admin/common/js/ui/time/DatePicker.ts

```diff
@@ -45,6 +52,15 @@ export class DatePicker
         if (builder.date) {
             this.setDate(builder.date);
         }
+
+        if (builder.defaultDate) {
+            this.setDefaultHandler(
+                () => {
+                    this.setSelectedDate(builder.defaultDate);
+                },
+                builder.defaultDate
+            );
```

Comment:
You don't need to pass both handler AND the value the handler sets. The handler is enough. And curly brackets in the handler are not needed.

if (builder.defaultDate) {
    this.setDefaultHandler(() => this.setSelectedDate(builder.defaultDate));
}